doto-backend/src/Constants/http-response.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    unauthorized: Number(401),
+    forbidden: Number(403),
+    successful: Number(200),
+    badRequest: Number(400),
+};
```

Comment:
Any reason for the Number() syntax? Can we just have 

```
module.exports = { 
    unauthorized: 401
}
```

Reply:
I also feel like the keys should be capital because http-response.js is acting as an Enum object.

Reply:
> Any reason for the Number() syntax? Can we just have
> 
> ```
> module.exports = { 
>     unauthorized: 401
> }
> ```

It wasn't working without it before so I looked up a solution which seemed to work if I had put the number syntax.